spark-3.1-spanner-lib/src/test/java/com/google/cloud/spark/spanner/SparkSpannerIntegrationTestBase.java

```diff
@@ -0,0 +1,71 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.cloud.spark.spanner;
+
+import org.apache.spark.sql.SparkSession;
+import org.junit.Before;
+import org.junit.ClassRule;
+import org.junit.rules.ExternalResource;
+
+public class SparkSpannerIntegrationTestBase extends SpannerTestBase {
+
+  @ClassRule public static SparkFactory sparkFactory = new SparkFactory();
+  @ClassRule public static TestDataset testDataset = new TestDataset();
+
+  protected SparkSession spark;
+  protected String testTable;
```

Comment:
The testTable is not used. Either remove this variable or setup the value to be compositeTable.

Reply:
Done, thanks @halio-g!